### DISCUSSION TITLE
A quick way to add/remove a custom tag?

### DISCUSSION THREAD

**Discussion by martin-braun** [kdheepak/taskwarrior-tui#504] [Ideas]
Exploring `taskwarrior-tui` right now. Really nice tool, the `t` binding is interesting. I'd like to propose to have a `T` binding which will prompt for a string (tag name). If the given string is not a tag on the current task, it should add the tag. If the given string already exists as tag, it should remove the tag. This would allow tag management without editing the entire task using `e`.

What do you think?

**Comment by kdheepak** [comment]
I like the idea! Thanks for making a discussion about it.

**Reply by niderhoff** [reply]
Currently I am doing that with https://kdheepak.com/taskwarrior-tui/configuration/advanced/#configure-user-defined-shortcuts but it takes some time to execute and will always show the terminal for a quick second and then open the tui again which is suboptimal.
